home_audit.node.js

```diff
@@ -1423,11 +1423,23 @@ let validationRules = {
         return TypeRules._int_or_zero(_homeValues.floor_area_1) + TypeRules._int_or_zero(_homeValues.floor_area_2);
     },
 
+    /*
+     * Get projected roof area
+     * @param {string} '1' or '2'
+     */
+    _get_proj_roof_area: function(value) {
+        const roofArea = TypeRules._int_or_zero(_homeValues['roof_area_'+value]);
+        return _homeValues['roof_type_'+value] === 'cath_ceiling'
+          ? roofArea * Math.cos(30 * (Math.PI/180))
+          : roofArea;
+    },
+
     /*
      * Get combined roof area
      */
     _get_combined_roof_area: function() {
```

Comment:
_get_combined_roof_area is also called from _check_combined_area on line 1502 and changed the validation logic there.  
BTW: I am missing something. I would expect _check_combined_area to fail for all homes with more than one story.